### ISSUE TITLE
Apa 7 websites

### ISSUE THREAD

**Issue by DrakeDragon5140** [citation-style-language/styles#5447]
It's my understanding that Mendeley, Zotero, etc use this repository, since Mendeley says to contact Github and Github says contact Mendeley, here I am... in text citations for websites are wrong! If I'm trying to cite the About page of the website blacklivesmatter.com, it does NOT cite the About (italicized), just the main site. Perfection matters in a dissertation and my Chair is getting tired of trying to figure out if I'm wrong, right or almost right! Can someone please fix websites in Apa 7?

**Comment by adam3smith** [comment]
Hard to follow this -- generally speaking, APA 7 is implemented correctly, and with lots of attention to detail, in CSL, including websites and webpages, so this is more likely than not an issue of how you have data entered in Mendeley, but we'd need to see an example to say more.

**Comment by DrakeDragon5140** [comment]
There ya go

**Comment by adam3smith** [comment]
If this was meant to include an attachment, those don't work by email to github. You actually have to come here and drag the file to your comment (or you paste the relevant example in your response text)

**Comment by DrakeDragon5140** [comment]
Seems like you should get support to fix that problem....


From Perdue OWL: If the resource was written by a group or organization,
use the name of the group/organization as the author. Additionally, if the
author and site name are the same, omit the site name from the citation.



Group name. (Year, Month Date). *Title of page*. Site name. URL

See this is correct: Black Lives Matter. (2021). *About*.
https://blacklivesmatter.com/about/



This is not: (Black Lives Matter, 2021) because it is a webpage and the
title of the page is About



Using the organization as the Author, like Perdue OWL says I can, It should
be (Black Lives Matter, *About*, 2021)





Generic webpage, no author (the 2 is so that I can differentiate between
them for this example)



This is correct:

*Black Lives Matter 2*. (2021). www.blacklivesmatter.com

(*Black Lives Matter 2*, 2021)

https://columbiacollege-ca.libguides.com/apa/websites



In-Text Quote

(Corporation/Group's Name, year, Section Name section, para. Paragraph
Number if more than one paragraph in section)

Example: (Canadian Cancer Society, 2013, Behavioural research section,
para. 2)

*Note: *When there are no visible page numbers or paragraph numbers, you
may cite the section heading and the paragraph number to identify where
your quote came from.



That is a clearer example of how it should be



Does this clear it up?

**Comment by adam3smith** [comment]
You're misunderstanding the OWL guide:

> (Black Lives Matter, 2020) 

Is the correct in-text citation. You're looking at the part of the guide that's about the reference list. The in-text citation in APA never has author _and_ title.

The second example is specifically about how to point to a location of a specific quote for a work without page numbers; that doesn't look relevant here at all.

**Comment by DrakeDragon5140** [comment]
It's a source without a page number, which means that About should be in
the citation but don't worry, I've contacted the people that actually write
the APA style and they say it needs the About. I can just tell all of them
that you're right and they're wrong, no worries. Thanks for looking at it.

**Comment by adam3smith** [comment]
Sure. If you do need help in the future, please find a way to communicate at basic levels of respect and politeness. We're doing this for free & have no interest in dealing with snark.
